Week2/homework/maartjes_work.js

```diff
@@ -44,4 +44,21 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+//Map the tasks to durations in hours----
+const minutesToHours = tasks.map(tasks => tasks.duration / 60);
+console.log(minutesToHours);
+
+//Filter out everything that took less than two hours---
```

Comment:
What you did really well throughout the exercise is making the variables descriptive. As a result many, if not all, the comments are not necessary to include. Especially if a project gets bigger, it becomes easier to read throughout the code and to maintain the code. Try to review this comment throughout this exercise.